patterns/cell-based-eks/0.vpc/main.tf

```diff
@@ -0,0 +1,47 @@
+provider "aws" {
```

Comment:
I don't believe the cluster-per-AZ design requires splitting up the Terraform configurations into multiple directories. We should collapse this back down to a single directory, but have multiple cluster definitions - one for each AZ used. This can be shown with a set of definitions split into multiple files - for example:
- `az1.tf`
- `az1.yaml`
- `az2.tf`
- `az2.yaml`
- `az3.tf`
- `az3.yaml`

Within each of these AZ specific Terraform files we'll have:
- EKS cluster definition
- Addons definition
- Kubernetes and Helm aliased providers scoped to that cluster and addon definition

Then within each of the AZ specific YAML files will be the Karpenter specific manifests for that AZ and cluster within that AZ.

Thoughts?

Reply:
Done, made changes as suggested. We were following the istio multi-cluster pattern structure before.